modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqTranslate.scala

```diff
@@ -0,0 +1,54 @@
+package edu.gemini.seqexec.server
+
+import edu.gemini.seqexec.engine.{Action, Result, Sequence, Step}
+import edu.gemini.seqexec.server.SeqexecFailure.{Execution, UnrecognizedInstrument}
+import edu.gemini.spModel.config2.{Config, ConfigSequence, ItemKey}
+import edu.gemini.spModel.obscomp.InstConstants._
+import edu.gemini.spModel.seqcomp.SeqConfigNames._
+
+import scalaz.\/
+import scalaz.NonEmptyList._
+import scalaz._
+import Scalaz._
+import scalaz.syntax.TraverseOps
+
+/**
+  * Created by jluhrs on 9/14/16.
+  */
+object SeqTranslate {
+
+  implicit def toAction[A](x: SeqAction[A]): Action = x.run map {
+    case -\/(e) => Result.Error(e)
+    case \/-(r) => Result.OK(r)
+  }
```

Comment:
Sure, I get that `SeqAction`s have to be converted to `Action`s, and that `toAction` is necessary.

I'm just trying to understand why `toAction` is an implicit function instead of an ordinary one. It's not that I'm against making `toAction` `implicit`, it's just me trying to understand Scala basics 😉 .